src/services/userService.js

```diff
@@ -38,7 +38,9 @@ function updateUser(id, name) {
   return newUser;
 };
 
-function normalize({ id, name }) {
+function normalize(user) {
```

Comment:
I meant that the whole function is redundant, you don't need to create a function to destruct an object